.github/workflows/tox.yml

```diff
@@ -27,7 +29,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        pip install "tox<4" tox-gh-actions tox-pip-version
+        pip install "tox<4" tox-gh-actions tox-pip-version setuptools-scm
```

Comment:
I *think* this addition is also not necessary, since we do pull in `setuptools-scm` via `setup.py` and `base.txt` now.

Reply:
I tried again, and even with `setuptools-scm` in `base.txt`, gh-actions was missing this and all tests failed. So, I think we need this addition here.

Reply:
I completely understand that I'm nitpicking here, but I *think* https://github.com/mrtmm/hastexo-xblock/pull/9 has a more correct fix for this. :) What do you think?

Reply:
Sure, looks good. I merged and squashed it, not sure why the PR didn't close, I think I should have pushed the branch before squashing it.